integration-tests/test_no_upload.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import glob
+import os
+import shutil
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_no_upload(insights_client):
```

Comment:
My suggestion is to move this test to test_client_options.py for the reason that it is one of the insights-client options. or if you plan to have more no_upload tests in this class then we can discuss.

Reply:
done